src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int STRING_INDEX = 0;
+    private static final int RESULT_INDEX = 1;
+    private static final String SUPPLY_FIELD = "supply";
+    private static final String BUY_FIELD = "buy";
+    private static final String RESULT_FIELD = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String readData = readReport(fromFileName);
+        String report = getReport(readData);
+        writeReport(toFileName, report);
+    }
+
+    private String readReport(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("I can`t read this file", e);
```

Comment:
exception msg must be informative
![image](https://user-images.githubusercontent.com/20326858/209204588-5d8b0255-0891-4b37-8b90-695b2f63e6c5.png)